configure.ac

```diff
@@ -54,6 +54,7 @@ AC_CHECK_HEADERS([sys/socket.h])
 AC_CHECK_HEADERS([sys/types.h])
 AC_CHECK_HEADERS([syslog.h])
 AC_CHECK_HEADERS([unistd.h])
+AC_CHECK_HEADERS([numa.h])
```

Comment:
Is there a pkgconfig file from libnuma? That might be nicer to check for, because the cflags and linker flags would be autodetected.

Reply:
There is numactl-devel (libnuma-dev for debian)